plugins/org.wso2.developerstudio.eclipse.esb.docker/plugin.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+  Version 2.0 (the "License"); you may not use this file except
+  in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied. See the License for the
+ specific language governing permissions and limitations
+ under the License.
+-->
+
+<?eclipse version="3.4"?>
+<plugin>
+    <extension point="org.eclipse.ui.popupMenus">
+        <objectContribution objectClass="org.eclipse.core.resources.IProject" 
+        id="org.wso2.developerstudio.eclipse.esb.docker.exportandgeneratedocker">
+            <action
+                label="Generate Docker Image"
+                icon="icons/car.png"
```

Comment:
please create a issue to add image to match with docker.

Reply:
https://github.com/wso2/devstudio-tooling-esb/issues/566